### ISSUE TITLE
Quickly navigate to line number in blame view

### ISSUE THREAD

**Issue by ilyagr** [extrawurst/gitui#2219]
**Is your feature request related to a problem? Please describe.**
When I open the blame view, I ahve to navigate to the piece of code I care about, often many lines down.

**Describe the solution you'd like**

A way to type in the line number to jump to (e.g. so I could type `Shift-L`, `151`, `<Enter>`).

**Describe alternatives you've considered**

Specifying the file (and line) in the command line, as suggested in https://github.com/extrawurst/gitui/issues/1382, would also be great.

**Comment by extrawurst** [comment]
Yeah I would like to have a general reusable popup to use this in a bunch of places
